src/link/Dwarf.zig

```diff
@@ -194,7 +194,7 @@ pub const DeclState = struct {
                     dbg_info_buffer.appendAssumeCapacity(0);
                     // DW.AT.type, DW.FORM.ref4
                     var index = dbg_info_buffer.items.len;
-                    try dbg_info_buffer.resize(index + 4);
+                    try dbg_info_buffer.writer().writeByteNTimes(0xaa, 4);
```

Comment:
Using 0xaa is generally bad practice because it is indistinguishable from actually undefined bytes, which was happening before. It would be better to use zeroes or a different sequence, so that if a programmer sees 0xaa in a debugger, they can guess that it is likely an undefined zig value, while if they see zeroes, or another value of your choosing, they can guess that it is likely an unresolved relocation.

My vote would be for zeroes, however, it should definitely not be 0xaa.

Reply:
Mhm, that's a good point - something useful for me might be extremely confusing for an actual dev/user.

Reply:
Last commit reverted, meaning we zero out unresolved relocs.